doc/user/content/get-started/install-materialize.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Install Materialize locally"
+description: "Download the Docker image and run Materialize locally for evaluation"
+menu:
+  main:
+    parent: "get-started"
+    weight: 16
+    name: "Install Materialize locally"
+
+---
+
+For testing/evaluation purposes, you can download the Materialize Docker image
+and run Materilize locally in a Docker container.  The Docker image:
+
+- Includes the [Quickstart](/get-started/quickstart) data source
+
+- Supports connecting your data sources
+
+- Provides [community support via
+  Slack](https://materializecommunity.slack.com/join/shared_invite/zt-2bad5ce4i-ZsiPWI5jd7Q9pRDGYj3dkw#/shared-invite/email)
+
+To more fully evaluate
+Materialize and its features, Materialize offers a [free trial
+account](https://materialize.com/register/?utm_campaign=General&utm_source=documentation).
+
+{{< warning >}}
+
+The Docker image is for <redb> testing/evaluation purposes only. It is not
+suitable for production deployments</redb>.  See also [Limitations](#limitations).
+
+{{</ warning >}}
+
+### License and Privacy Policy
+
+- Use of the Docker image is subject to Materialize's [BSL
+  License](https://github.com/MaterializeInc/materialize/blob/main/LICENSE).
+
+- By downloading the Docker image, you are agreeing to Materialize's [privacy
+  policy](https://materialize.com/privacy-policy/).
+
+### Prerequisites
+
+- Docker. If [Docker](https://www.docker.com/) is not installed, refer to its
+[official documentation](https://docs.docker.com/get-docker/) to install.
+
+### Run Materialize in a Docker container
+
+{{< note >}}
+
+- Use of the Docker image is subject to Materialize's [BSL
+   License](https://github.com/MaterializeInc/materialize/blob/main/LICENSE).
+
+- By downloading the Docker image, you are agreeing to Materialize's [privacy
+   policy](https://materialize.com/privacy-policy/).
+
+{{</ note >}}
+
+1. In a terminal, issue the following command to run a Docker container from the
+   Materialize image. The command downloads the Materialize image, if one has
+   not been already downloaded.
+
+   ```sh
+   docker run -v mzdata:/mzdata -p 6875:6875 -p 6876:6876 materialize/materialized
+   ```
+
+   When running locally:
+
+   - Materialize logs are emitted to the `stderr`.
+   - The SQL interface is available on port `6875`.
+   - The HTTP interface is available on port `6876`.
+   - A default user `materialize` is created.
+   - A default database `materialize` is created.
+
+1. Connect to Materialize using your preferred SQL client, using the following
+   connection field values:
+
+   | Field    | Value         |
+   |----------|---------------|
+   | Server   | `localhost`   |
+   | Database | `materialize` |
+   | Port     | `6875`        |
+   | Username | `materialize` |
+
+   For example, if using `psql`
```

Comment:
```suggestion
   For example, if using `psql`:
```

Reply:
@ggnall, reminder for the future that we should push users to `mz`. Also, @kay-kim: we used to provide instructions to install `psql`, because most users did not have it installed. You can point people to the [SQL clients page](https://materialize.com/docs/integrations/sql-clients/#psql).

<img width="827" alt="Screenshot 2024-08-19 at 17 48 30" src="https://github.com/user-attachments/assets/9a5b87a4-0ba1-4696-b46c-cf783b78622b">

Reply:
👍  I'll link to integrations/sql-clients/#psql. 
Since people are downloading for their eval, I'm hoping (recognizing hope and reality are 2 different things) they'll connect with the client they have on hand and just use the example statement really as just one example as opposed to a suggestion.  (but, definitely, doubly-recognizing that the reality in practice may differ).